apps/fluent-tester/src/TestComponents/Switch/Switch.tsx

```diff
@@ -18,8 +19,12 @@ const styles = StyleSheet.create({
 });
 
 const StandardUsage: React.FunctionComponent = () => {
+  const memoizedStyles = React.useMemo(
+    () => [Platform.OS === 'android' ? { ...commonTestStyles.androidContainer, height: 180 } : { ...commonTestStyles.settingsPicker }],
+    [],
+  );
```

Comment:
Could you point out in screenshots what doesn't look nice? :) 

In general, If it's a mobile specific styling problem, I'd prefer mobile specific styles that also encompass iOS. Similar to what we do in `FluentTester.tsx`. So yes, if it can be extended, that would be great :)

Reply:
**Before the Mobile Specific Styling :** 

![nw1](https://user-images.githubusercontent.com/30728574/205841583-9a4b7023-bb77-41fe-8059-5a9095e5d154.png)
![nw2](https://user-images.githubusercontent.com/30728574/205841588-6e5b68b1-a5ae-4785-813e-6d54cc178368.png)


**After the Mobile Specific Styling :** 

![w2](https://user-images.githubusercontent.com/30728574/205841563-65f697c1-1fcf-4929-8a5b-a5035d27eac7.png)
![w3](https://user-images.githubusercontent.com/30728574/205841568-934f131d-2c5c-47c2-ae07-11da82fadb6c.png)
![w1](https://user-images.githubusercontent.com/30728574/205841571-637ab3a2-0208-4ef0-a38a-835ccbe1f6d8.png)




@Saadnajmi , As mentioned above : 
As few tokens are required on android for Switch for eg : consider the switch has default margin on win32 but for android there is no default margin as per design , so elements were not very clean among the sections.

Reply:
Screenshots added.